publicodes/docs/api.md

```diff
@@ -10,14 +10,15 @@ Crée un moteur d'évaluation avec les règles publicodes données en argument.
 
 -   `rules` : les règles publicodes utilisée. Ces dernières peuvent-être sous la
     forme d'une chaine de caractère `yaml` publicodes (ou d'un object javascript
-    correspondant). Elle peuvent aussi être sous la forme de règles publicodes déjà parsées (voir [`parsePublicode`](#parsePublicode))
+    correspondant). Elle peuvent aussi être sous la forme de règles publicodes déjà parsées
 
 **Retourne**
 Un moteur d'évaluation qui expose les fonctions suivantes :
 
 -   setSituation
 -   evaluate
--   getParsedRules
+-   getRules
```

Comment:
```suggestion
-   getParsedRules
```

Voir ne pas référencer cette méthode du tout, en attendant d'identifier nos différents cas d'usage et de définir une meilleure API publique

Reply:
Je compte 13 utilisations de `getParsedRules`. Le plus souvent pour récupérer la liste des noms `Object.keys(engine.getParsedRules())` ou vérifier qu'une règle existe `dottedName in engine.getParsedRules()`. Dans ces deux cas on pourrait utiliser une API `engine.getNames()` qui retourne la liste de tous les noms (à voir si on exclut les "virtualRules" où si on laisse une option).